README.md

```diff
@@ -19,99 +19,53 @@ You can watch videos of the system being developed on YouTube:
-* OpenGL 1.x compatible library (LibGL)
-* GUI toolkit (LibGUI)
-* Cross-process communication library (LibIPC)
-* HTML/CSS engine (LibWeb)
-* JavaScript engine (LibJS)
-* Markdown (LibMarkdown)
-* Audio (LibAudio)
-* Digital Signal Processing/Synthesizer Chains (LibDSP)
-* PCI database (LibPCIDB)
-* Terminal emulation (LibVT)
-* Out-of-process network protocol I/O (LibProtocol)
-* Mathematical functions (LibM)
-* ELF file handling (LibELF)
-* POSIX threading (LibPthread)
-* Higher-level threading (LibThreading)
-* Transport Layer Security (LibTLS)
-* HTTP and HTTPS (LibHTTP)
-* IMAP (LibIMAP)
-
-## Userland features
-
-* Unix-like libc and userland
-* Shell with pipes and I/O redirection
-* On-line help system (both terminal and GUI variants)
-* Web browser (Browser)
-* C++ IDE (HackStudio)
-* Desktop synthesizer (Piano)
-* E-mail client (Mail)
-* Various desktop apps & games
-* Color themes
+## Features
+* Modern x86 32-bit and 64-bit kernel with pre-emptive multi-threading
+* [Browser](Userland/Applications/Browser/) with JavaScript, WASM and more (check the spec compliance [here](https://libjs.dev/test262/), [here](https://css.tobyase.de/) and [here](https://libjs.dev/wasm/))
+* Security features (hardware protections, userland isolation, W^X memory, `pledge` & `unveil`, (K)ASLR, OOM-resistance, web-content isolation, state-of-the-art TLS algorithms, ...)
+* [System services](Userland/Services/) (WindowServer, LoginServer, AudioServer, WebServer, RequestServer, CrashServer, ...) and modern IPC
+* Good POSIX compatibility ([LibC](Userland/Libraries/LibC/), Shell, syscalls, signals, pseudoterminals, filesystem notifications, standard Unix [utilities](Userland/Utilities/), ...)
+* POSIX-like virtual file systems (/proc, /dev, /sys, /tmp, ...) and ext2 file system
+* Profiling, debugging and other development tools (Kernel-supported profiling, UserspaceEmulator, CrashReporter, interactive GUI playground, HexEditor, HackStudio IDE for C++ and more)
+* [Libraries](Userland/Libraries/) for everything from cryptography to OpenGL, audio, JavaScript, GUI, playing chess, ...
+* Support for many common and uncommon file formats (PNG, JPEG, GIF, MP3, WAV, FLAC, ZIP, TAR, PDF, ...)
+* Unified style and design philosophy, extensible theming system, [custom (bitmap and vector) fonts](https://fonts.serenityos.net/font-family)
+* [Games](Userland/Games/) (Solitaire, Minesweeper, 2048, chess, Conway's Game of Life, ...) and [demos](Userland/Demos/) (CatDog, Starfield, Eyes, mandelbrot set, WidgetGallery, ...)
+* Every-day GUI programs and utilities (Spreadsheet with JavaScript, TextEditor, Terminal, PixelPaint, various multimedia viewers and players, Mail, Assistant, Calculator, ...)
+
+... and all of the above are right in this repository, no extra dependencies, built from-scratch by us :^)
+
+Additionally, there are [over two hundred ports of popular open-source software](Ports/), including games, compilers, unix tools, multimedia apps and more.
```

Comment:
Wouldn't a link to Ports/AvailablePorts.md make more sense here because the sentence focuses on how many and what kind of ports, not the tech behind it?

Reply:
@linusg apparently has a Ports README rework in progress, so I'd say it's fine

Reply:
???

Reply:
nvm let's link AvailablePorts